98Installation/sec00virtualbox.md

```diff
@@ -8,22 +8,23 @@ title: VirtualBox
 
 We will be running a linux virtual machine using VirtualBox software on either your laptop, or if you do not have 
 one on one of the Desktop@UCL machines in Physics Lab 1. If you have a laptop this is the preferred option as it 
-will mean you will be able to work on the homework problems and coursework without needing access to a cluster room. 
-If you don’t have a laptop then that is fine as well as the whole course can be completed using the Desktop@UCL machines.
+will mean you will be able to work on the homework problems and coursework without needing access to a cluster room. If you don’t have a laptop then that is fine as well as the whole course can be completed using the Desktop@UCL machines.
+
+Download and install on your laptop:
+* VirtualBox from [virtualbox.org](https://www.virtualbox.org/wiki/Downloads) 
+* The VirtualBox Extension Pack (from same page, make sure the versions match 
```

Comment:
This sentence is either missing some text or a closing parenthesis.

Reply:
Also, the download page doesn't seem to give the option to choose an extension pack for a different version

Reply:
Text has been updated